src/Template/Pages/Modules/view.twig

```diff
@@ -21,6 +21,10 @@
             'id': 'form-main',
         })|raw }}
 
+
+            {# Init `_jsonKeys` data #}
+            {{ write_config('_jsonKeys', []) }}
```

Comment:
Il problema del layout a colonne che si rompe deriva da questa riga: l'istruzione torna un 1 che sta prima delle colonne e spagina. Dante propone per semplicità di assegnare il ritorno a un a var dummy.
I.e. `{% set a = write_config('_jsonKeys', []) %}`

Reply:
:scream: 
ok, meglio modificare `write_config` to not display anything :wink: 
I'll do that...

Reply:
fixed